.github/workflows/spelling2.yml

```diff
@@ -123,12 +160,38 @@ jobs:
     runs-on: ubuntu-latest
     needs: spelling
     permissions:
+      contents: read
       pull-requests: write
     if: (success() || failure()) && needs.spelling.outputs.followup && contains(github.event_name, 'pull_request')
     steps:
     - name: comment
-      uses: check-spelling/check-spelling@v0.0.21
+      uses: check-spelling/check-spelling@v0.0.22
       with:
         checkout: true
-        spell_check_this: check-spelling/spell-check-this@prerelease
+        spell_check_this: microsoft/terminal@main
         task: ${{ needs.spelling.outputs.followup }}
+        experimental_apply_changes_via_bot: ${{ github.repository_owner != 'microsoft' && 1 }}
+
+  update:
+    name: Update PR
+    permissions:
+      contents: write
+      pull-requests: write
+      actions: read
+    runs-on: ubuntu-latest
+    if: ${{
+        github.repository_owner != 'microsoft' &&
```

Comment:
talking to check-spelling is designed to be used in PRs forks -- not in the main repository (and it only works for PRs within a single repository)